tests/NuGet.Services.BasicSearchTests/StartupTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using NuGet.Services.BasicSearchTests.Fixtures;
+using NuGet.Services.BasicSearchTests.Models;
+using Xunit;
+
+namespace NuGet.Services.BasicSearchTests
+{
+    public class StartupTests : IClassFixture<StartupTestsFixture>
+    {
+        private readonly StartupTestsFixture _fixture;
+
+        public StartupTests(StartupTestsFixture fixture)
+        {
+            _fixture = fixture;
+        }
```

Comment:
We may want to set the packageVersion at startup, build lucene, and let service refresh. In this way we could make sure lucene is clean and won't affect other tests.